doc/examples-getting-started.rst

```diff
@@ -1,8 +1,8 @@
 ##############################
-Simple circuits and algorithms
+Getting started
 ##############################
 
-Learn more about working with advanced circuits and algoritms.
+Get started with Amazon Braket with some introductory examples.
```

Comment:
Maybe Get started with Amazon Braket: introductory examples. The double with in the sentence is a little weird.